### ISSUE TITLE
I lost my heirloom

### ISSUE THREAD

**Issue by Valdanif** [dalaranwow/dalaran-wow#4855]
Hello, like a week ago I deleted a hunter character with 2 heirlooms, the Champion Herod´s Shoulder and the Champion´s Deathdealer Breastplate. I didn´t remember that I had them so I deleted the character, when I deleted it, it didn´t pop me confirmation or anything related to my heirloom. Days later I realized that that character had heirlooms, and I want to ask you please if you could help me recover those heirlooms?, I spent real money for them. I would appreciate it.
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Edzero** [comment]
Making a ticket is the proper way to report this since it isn't a bug.

**Comment by ViolentJarvis** [comment]
If you make a ticket on Kirin-Tor it goes unanswered.

**Comment by TeridanDW** [comment]
> Making a ticket is the proper way to report this since it isn't a bug.
